runner/src/server/plugins/engine/models/WebhookModel.ts

```diff
@@ -20,10 +20,13 @@ export function WebhookModel(model: AdapterFormModel, state: FormSubmissionState
 
     let questions;
 
-    const {relevantPages} = model.getRelevantPages(state);
+    //Getting pages based on the answers in the state & this will allow the user to save all the given answers in the db
+    //TODO probable redesign will happen with https://dluhcdigital.atlassian.net/browse/FS-4341 Jira this tries to manage application state based on scenarios
+    const {relevantPages} = model.getRelevantPagesBasedOnState(state);
 
     questions = relevantPages.map((page) => pagesToQuestions(page, state));
     const fees = FeesModel(model, state);
+    console.log(`[WebhookModel] save data ${JSON.stringify(questions)}`);
```

Comment:
Remove or proper log message please

Reply:
I will remove this since this was added to monitor data